python/lsst/ip/diffim/psfMatch.py

```diff
@@ -376,25 +376,26 @@ def setDefaults(self):
 
     alardNGauss = pexConfig.Field(
         dtype=int,
-        doc="Number of Gaussians in alard-lupton basis",
+        doc="Number of base Gaussians in alard-lupton kernel basis function generation.",
         default=3,
         check=lambda x: x >= 1
     )
     alardDegGauss = pexConfig.ListField(
         dtype=int,
-        doc="Polynomial order of spatial modification of Gaussian basis functions. "
+        doc="Polynomial order of spatial modification of base Gaussians. "
             "List length must be `alardNGauss`.",
         default=(4, 2, 2),
     )
     alardSigGauss = pexConfig.ListField(
         dtype=float,
-        doc="Default sigma values in pixels of Gaussian kernel base functions. "
+        doc="Default sigma values in pixels of base Gaussians. "
             "List length must be `alardNGauss`.",
         default=(0.7, 1.5, 3.0),
     )
     alardGaussBeta = pexConfig.Field(
         dtype=float,
-        doc="""Multiplier of Gaussian sigmas between adjacent elements of the kernel basis list.""",
+        doc="Scaling multiplier of base Gaussian sigmas for automated sigma determination "
+            "(scaleByFwhm = True case).",
```

Comment:
scaleByFwhm should be in back-ticks, like `alardNGauss` in the entry above. I would also write this as "Used if `scaleByFwhm`=True". Or, something like that if that's inaccurate.

Reply:
Reworded.